quelpa.el

```diff
@@ -1806,6 +1806,22 @@ the `quelpa' command has been run in the current Emacs session."
                 (quelpa item)))
             quelpa-cache))))
 
+;;;###autoload
+(defun quelpa-upgrade (rcp)
+  "Upgrade a package found in `quelpa-cache'.
+RCP is a melpa recipe (list)."
+  (interactive
+   (list (when (quelpa-setup-p)
+           (let* ((quelpa-melpa-recipe-stores `(,quelpa-cache))
+                  (name (quelpa-interactive-candidate)))
+             (assq name quelpa-cache)))))
```

Comment:
Is `assq` the right thing to use here?  It uses `eq` for comparison, which usually isn't correct.  `assoc` or `alist-get` are more commonly used.

Reply:
I will use `assoc`, `alist-get` is introduced in Emacs 25.1 and we need this package to work with 24.3